auth/src/main/java/com/redhat/lightblue/rest/auth/ldap/InitialLdapContextProvider.java

```diff
@@ -33,6 +32,11 @@ public InitialLdapContextProvider(String server, String bindDn, String bindDNPwd
         }
         env.put(Context.INITIAL_CONTEXT_FACTORY, "com.sun.jndi.ldap.LdapCtxFactory");
         env.put(Context.PROVIDER_URL, server);
+        if(useSSL) {
+            System.setProperty("javax.net.ssl.trustStore", trustStore);
+            System.setProperty("javax.net.ssl.trustStorePassword", trustStorePassword);
```

Comment:
The built-in Java LDAP support uses these when you connect to an ldaps:// endpoint.  Using System.setProperty is totally tacky, which is why I opened https://github.com/lightblue-platform/lightblue-rest/issues/257.  However, we don't have time for all that at the moment...